onnx/backend/test/case/node/image_decoder.py

```diff
@@ -39,22 +42,32 @@ def generate_checkerboard(width, height, square_size):
     return image
 
 
-def generate_test_data(extension, pixel_format="RGB", h=40, w=40, tile_sz=5):
-    data, output = None, None
+def _generate_test_data(
+    format_: str,
+    pixel_format: str = "RGB",
+    h: int = 40,
+    w: int = 40,
```

Comment:
Could you please use 32x32 instead of 40x40 to create test data from Pillow? (just like what https://github.com/onnx/onnx/pull/5511 did) In that case I don't need to change the data again in my PR (https://github.com/onnx/onnx/pull/5511) and just focus on introducing python file there for frozen data.